tests/Application/.env

```diff
@@ -12,7 +12,7 @@ APP_SECRET=EDITME
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
-DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.5
+DATABASE_URL=mysql://root@127.0.0.1/sylius_brand_plugin_%kernel.environment%?serverVersion=5.5
```

Comment:
Any reason to change this? I know it's a small change :)

Reply:
I believe it should have unique name. Reasons:
- To easy switch between projects without any impact
- To not accidentally rewrite some sylius application database (as Sylius standard have same database name by default) when we run `composer try` or any other plugin-related things that impact database.
- In other projects that using `dev.db` - I also replacing to `root@127.0.0.1` to make `bin/console doctrine:database:create --if-not-exists` work properly for `composer ensure-database-created` script. As file-based database failing on option `--if-not-exists`.

Reply:
Okay, that's cool :)